practices/managing-static-assets.md

```diff
@@ -0,0 +1,13 @@
+# Managing static assets
+
+Static assets are files that don't need to be generated, modified or processed by an application before being delivered to the user. This typically includes CSS, JavaScript, images and web fonts. Static assets change rarely, and their content doesn't depend on user input or preferences.
+
+We host on our own infrastructure these kinds of static resources, instead of including them on our pages from an third-party CDN. This applies especially to JavaScript files or libraries, and font files.
```

Comment:
```suggestion
We host these kinds of static resources on our own infrastructure & CDN, instead of including them on our pages from other peoples infrastructure & CDN. This applies especially to JavaScript files or libraries, and font files.
```

I think this is an important clarification -- at first read I understood this line to mean "do not host on any CDNs (even those we control)"

Reply:
Doesn't "third-parties infrastructure and CDN" sound better than "other peoples infrastructure & CDN"? Especially considering that joke about "the cloud is just other people's computers"?

I'm wondering how can I make it clear that I mean "our server and CDN" versus "random-js-library-cdn.com" without having to mention any of them by name.

Reply:
i dont think so no, "our" and "others" are opposites.